packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -1572,8 +1786,10 @@ describe("database test", async () => {
         };
         await addOneEService(eService1, postgresDB, eservices);
 
-        const conflictEServiceName = `${eService1.name} - clone - 1/1/2024 12:00:00`;
-
+        const cloneTimeStamp = new Date();
+        const expectedLocalDate = cloneTimeStamp.toLocaleDateString("it-IT");
+        const expectedLocalTime = cloneTimeStamp.toLocaleTimeString("it-IT");
+        const conflictEServiceName = `${eService1.name} - clone - ${expectedLocalDate} ${expectedLocalTime}`;
```

Comment:
Could we revert this change? Same as above.

Reply:
Same as above